connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -411,7 +412,7 @@ public static String getClientId() {
 
     public static boolean showMobileDataDialog() {
```

Comment:
I guess method also needs to be renamed.

Reply:
Fixed in e3fa8e2.

Reply:
Good catch.